getting_started_modules.md

```diff
@@ -330,4 +330,9 @@ When you re-run your code using `terraform apply`, the environment variables cre
 > :bulb: Note that you can use [Vault plugin](https://www.vaultproject.io/docs/internals/plugins) to get more useful functionallity out of Vault.
 
 #### 7. CI/CD Pipeline To Continuously Test The Module When Changes Are Made
-TODO
+To test repositories you can use [github actions](https://github.com/features/actions). 
+They are written as workflows, which you can find under [.github/workflows](/.github
+/workflows).
```

Comment:
one line
```suggestion
They are written as workflows, which you can find under [.github/workflows](/.github/workflows).
```

Reply:
Sonofa... Okokok, I'll end this once and for all. And thank you

Reply:
Done